.github/workflows/iic2513-node.yml

```
@@ -4,21 +4,44 @@ on: pull_request
 
 jobs:
   tests:
+    name: Tests and linters
+    timeout-minutes: 10
     runs-on: ubuntu-latest
-
-    timeout-minutes: 12
-
+    env:
+      DATABASE_URL: postgresql://localhost/postgres
+      DB_DIALECT: postgres
+      DB_USERNAME: postgres
+      DB_PASSWORD: password
+      DB_NAME: iic2513template_test
+    services:
+      db:
+        env:
+          POSTGRES_HOST_AUTH_METHOD: trust
+        image: postgres:12.4
+        ports: ['5432:5432']
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    
     strategy:
       matrix:
         node-version: [ 12.x ]
-
+        
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v2
-      with:
-        node-version: ${{ matrix.node-version }}
-    - name: Install dependencies
-      run: yarn install
-    - name: Run ESLint
-      run: yarn lint
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: yarn install
+      - name: Setup Database
+        run: |
+          yarn sequelize db:create
+          yarn sequelize db:migrate
+      - name: Run ESLint
+        run: yarn lint
+      - name: Run jest tests
+        run: yarn test
```

Esto lo correría con la opción `--silent`. En el check puedes ver que aparece mucho output actualmente (muchos `console.log` de Sequelize)
```suggestion
        run: yarn test --silent
```